scripts/prebuild-fix.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/zsh
```

Comment:
By whom it will be executed?

Reply:
MacOS users. Apple replaces bash with zsh as the default shell in macOS Catalina.